src/java.base/share/classes/com/sun/crypto/provider/GCTR.java

```diff
@@ -139,4 +139,102 @@ int doFinal(byte[] in, int inOfs, int inLen, byte[] out,
         }
         return inLen;
     }
-}
+
+    // Maximum buffer size rotating ByteBuffer->byte[] intrinsic copy
+    private static final int MAX_LEN = 1024;
+
+    int doFinal(ByteBuffer src, ByteBuffer dst) throws IllegalBlockSizeException {
```

Comment:
Conventionally, we put doFinal() method after update() method.

Reply:
ok